typescript/quicksight/README.md

```diff
@@ -0,0 +1,54 @@
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Cfn-Only](https://img.shields.io/badge/stability-Cfn--Only-important.svg?style=for-the-badge)
+
+> **This is an experimental example. It may not build out of the box**
+>
+> This example is built on Cfn resources.
+>
+> It requires additional infrastructure prerequisites that must be created before successful build, see below.
+>
+> If build is unsuccessful, please create an [issue](https://github.com/aws-samples/aws-cdk-examples/issues/new) so that we may debug the problem
+---
+<!--END STABILITY BANNER-->
+
+## Overview
+
+This project demonstrates how to set up Amazon Quicksight. It will set up a S3 Bucket, import some test data and makes it available as a datasource to quicksight.
+
+With this setup you can create analysis in the console to view e.g. the world-population data:
+world-population.csv file in data directory taken from https://data.worldbank.org/indicator/SP.POP.TOTL
+License: CC BY-4.0
+
+## Build
+ 
+To build this app, you need to be in this example's root folder. Then run the following:
+ 
+```bash
+npm install -g aws-cdk
+npm install
+npm run build
+```
+
+## Resources
+
+1. * `Bucket` (aws-s3): Creates an Amazon S3 bucket with encryption and full access control, used to store data files and the manifest for QuickSight.
+2. * `BucketDeployment` (aws-s3-deployment): Deploys world population data and manifest JSON files to the S3 bucket, making them accessible for analysis in QuickSight.
+3. * `CfnDataSource` (aws-quicksight): Defines an Amazon QuickSight data source that connects to the S3 bucket to access the uploaded CSV files.
+4. * `CfnDataSet` (aws-quicksight): Configures a QuickSight dataset that organizes and structures the CSV data from the S3 bucket for reporting and analysis.
+5. * `CfnManagedPolicy` (aws-iam): Creates an IAM policy granting permissions to the QuickSight service role, allowing access to the S3 bucket and other necessary actions.
+
+## Deploy
+
+1. Create a quicksight account [by following these instructions](https://docs.aws.amazon.com/quicksight/latest/user/signing-up.html).
+2. Use the arn of the quicksight account and pass it to cdk using a context `cdk deploy --context quicksightAccountArn=<arn>`
```

Comment:
What is this account? Is it a service account? The ARN of the AWS account? Can this be precalculated?

Can you give and example of the format?

Reply:
The ARN can't be precalculated, I added an example and a bit more of an explanation of what kind of values need to be put into the ARN.